patroni/postgresql/rewind.py

```diff
@@ -59,11 +59,10 @@ def trigger_check_diverged_lsn(self):
         if self.can_rewind_or_reinitialize_allowed and self._state != REWIND_STATUS.NEED:
             self._state = REWIND_STATUS.CHECK
 
-    def check_leader_is_not_in_recovery(self, **kwargs):
-        if not kwargs.get('database'):
-            kwargs['database'] = self._postgresql.database
+    @staticmethod
+    def check_leader_is_not_in_recovery(conn_kwargs):
```

Comment:
```suggestion
    def check_leader_is_not_in_recovery(conn_kwargs={}):
```
add default? So you don't have to call it like `check_leader_is_not_in_recovery({})`

Reply:
No, it is supposed to always get some connection parameters.
The empty `dict` is only passed from the unit-tests.